contrib/broker/k8s/Dockerfile

```diff
@@ -20,13 +20,16 @@ RUN export DEBIAN_FRONTEND=noninteractive \
  && apt-get update \
  && apt-get install -y \
     ca-certificates \
+    curl \
+ && curl -Ls -o /tmp/helm-${HELM_VERSION}-linux-amd64.tar.gz \
+    http://storage.googleapis.com/kubernetes-helm/helm-${HELM_VERSION}-linux-amd64.tar.gz \
+ && mkdir -p /opt/services \
+ && tar -C /opt/services -x -z -f /tmp/helm-${HELM_VERSION}-linux-amd64.tar.gz \
+    --strip-components=1 linux-amd64/helm \
+ # cleanup
```

Comment:
```
[kent@mbp k8s]$ docker -v
Docker version 1.12.2, build bb80604
```

The output shows everything ran as expected:

```
[kent@mbp k8s]$ make docker
Building k8s-broker for linux
Building Docker
Sending build context to Docker daemon 68.63 MB
Step 1 : FROM debian
 ---> 73e72bf822ca
Step 2 : ENV HELM_VERSION v2.0.0-rc.2
 ---> Using cache
 ---> f1e5608a0de5
Step 3 : RUN export DEBIAN_FRONTEND=noninteractive  && apt-get update  && apt-get install -y     ca-certificates     curl  && curl -Ls -o /tmp/helm-${HELM_VERSION}-linux-amd64.tar.gz     http://storage.googleapis.com/kubernetes-helm/helm-${HELM_VERSION}-linux-amd64.tar.gz  && mkdir -p /opt/services  && tar -C /opt/services -x -z -f /tmp/helm-${HELM_VERSION}-linux-amd64.tar.gz     --strip-components=1 linux-amd64/helm  && rm -f /tmp/helm-${HELM_VERSION}-linux-amd64.tar.gz  && apt-get purge -y --auto-remove curl  && apt-get clean  && rm -rf /var/lib/apt/lists/*
 ---> Using cache
 ---> 5938208989cc
Step 4 : ENTRYPOINT /opt/services/k8s-broker --helm_binary /opt/services/helm
 ---> Using cache
 ---> 6751815d501c
Successfully built 6751815d501c
```

Also note that if the comment _were_ problematic, the `Dockerfile` would fail to compile because Docker would expect a directive on line 30 and would find some bash nonsense instead.

So unless this behavior has really, truly changed in a recent version of Docker, there is no problem here.